server/repository/src/db_diesel/user_permission_row.rs

```diff
@@ -105,9 +105,19 @@ impl<'a> UserPermissionRowRepository<'a> {
 
     pub fn delete_by_user_id(&self, user_id: &str) -> Result<(), RepositoryError> {
```

Comment:
Think a better name would be good here, e.g. `delete_basic_permissions_by_user_id()`.

Or even better have a `UserPermissionRepository::delete(filter)` method. Do we do this somewhere already? hopefully straight forward to implement?